DirectProgramming/DPC++/ParallelPatterns/dpc_reduce/src/main.cpp

```diff
@@ -316,7 +320,6 @@ float calc_pi_dpstd_native3(size_t num_steps, int groups, Policy&& policy) {
   auto calc_end = oneapi::dpl::end(buf);
```

Comment:
I see `calc_begin`, `calc_end` are not used in this function. It also correct for the `calc_pi_dpstd_native4` function

Reply:
will fix